management/dbaas/helpers.go

```diff
@@ -0,0 +1,44 @@
+package dbaas
+
+import (
+	"testing"
+
+	dbaasClient "github.com/percona/pmm/api/managementpb/dbaas/json/client"
+	"github.com/percona/pmm/api/managementpb/dbaas/json/client/kubernetes"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	pmmapitests "github.com/Percona-Lab/pmm-api-tests"
+)
+
+func registerKubernetesCluster(t *testing.T, kubernetesClusterName string) {
```

Comment:
We might want to use `t.Cleanup` to call unregisterKubernetesCluster on test exit instead of adding `defer unregisterKubernetesCluster` everywhere

Reply:
:+1:

Reply:
The whole point of t.Cleanup is that it can called _there_.